io.openliberty.java.internal_fat_22/README.md

```diff
@@ -0,0 +1,89 @@
+# Repo for quick testing of Liberty apps
+Especially useful for new Java versions
+
+### For versions of Java that already have gradle support
+
+To build the WAR file locally, from the root directory of the open-liberty-misc repository run:
+
+```
+./gradlew io.openliberty.java.internal_fat_22:build
+```
+
+To build+run locally, this currently requires some manual effort:
+* Note, the **build.gradle** file is configured to look for a Liberty install under `/wlp`. You might need to change it to point it to your local test Liberty install instead.
+* Copy the `jvm.options` file and the contents of the `server.env` file from the project's `run` directory and add them to your `/wlp/usr/servers/io.openliberty.java.internal_fat_22Server/server.env` file (or equivalent location).
```

Comment:
This also seems like it could be automated if we were guaranteed to have a /wlp directory

Reply:
I am going to opt instead to clean up the README.md file instead to remove the testing pieces and save that for extra credit.  :-)